### PR TITLE
Show banner if cookies are disabled

### LINKED ISSUE #1209: Show alert if cookies are disabled
re: https://trello.com/c/hcVloaHo/17-user-understands-what-limitations-they-have-if-their-cookies-are-disabled-and-how-to-turn-them-on

detect if cookies are disabled. If so,
- notify user of what the consequences are (possibly a banner)
- save this info to redux

### PR DESCRIPTION
**Pull request by sastels**
resolves #1209

### CODE REVIEW

**Comment by sastels** [comment]
[User understands what limitations they have if their cookies are disabled and how to turn them on](https://trello.com/c/hcVloaHo/17-user-understands-what-limitations-they-have-if-their-cookies-are-disabled-and-how-to-turn-them-on)

**Comment by smcmurtry** [comment]
Looks like it's working fine and the banner looks great. Only issue is that it's in the wrong place on both the benefits-directory and the favourites page:

Benefits directory design: https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b758c1e04618a550dec9617

Favourites design: https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b562e457ff49a75561f23b8

File: utils/common.js [hunk 1]
```diff
@@ -42,3 +42,17 @@ export const showQuestion = (question_variable_name, index, reduxState) => {
...
+  try {
+    // Create cookie
+    document.cookie = "cookietest=1";
+    const cookiesDisabled = document.cookie.indexOf("cookietest=") === -1;
+    // Delete cookie
+    document.cookie = "cookietest=1; expires=Thu, 01-Jan-1970 00:00:01 GMT";
+    return cookiesDisabled;
+  } catch (e) {
+    return true;
+  }
```
Comment (smcmurtry) [inline utils/common.js:57]: Cool!

**Review by smcmurtry — APPROVED** [review]
Great work Steve!

**Pull request merged**